@xen-orchestra/backups/_runners/_writers/_MixinXapiWriter.js

```diff
@@ -28,9 +43,23 @@ exports.MixinXapiWriter = (BaseClass = Object) =>
           let clonedVm
           try {
             const baseVm = xapi.getObject(this._targetVmRef) ?? (await xapi.waitObject(this._targetVmRef))
-            const clonedRef = await xapi
-              .callAsync('VM.clone', this._targetVmRef, `Health Check - ${baseVm.name_label}`)
-              .then(extractOpaqueRef)
+            let clonedRef
```

Comment:
Is ```cloneRef``` still a relevant variable name, since it can now be either a clone ref or a copy ref? 🤔